src/lib/network/wicked.rb

```diff
@@ -85,5 +86,14 @@ def query_wicked(iface, query)
     def ibft_interfaces
       Yast::Execute.stdout.locally!(IBFT_CMD, "-l").gsub("\n", " ").split(" ").uniq
     end
+
+    # Returns an array of interface names which are configured via firmware
+    #
+    # @return [Array <String>] array of interface names
+    def firmware_interfaces
+      output = Yast::Execute.stdout.locally!(WICKED_PATH, "firmware", "interfaces")
+      interfaces = output.gsub(/^(ibft|nbft|redfish)/, "").gsub("\n", " ").split(" ")
```

Comment:
Wouldn't be better to split by `/\s/` just in case they start to use tabs?

Reply:
Thinking it twice I see another problem (not serious but...). What if tomorrow they invent a new firmware technology that is not ibft, nbft or redfish?

Maybe we should rely more in the format (ignore the fist column and catch the rest) than in the content (check for the technology names).